run.go

```diff
@@ -77,6 +77,10 @@ var runCommand = &cli.Command{
 			Usage: "Set custom DNS servers (only meaningful for \"bridge\" network)",
 			Value: cli.NewStringSlice("8.8.8.8", "1.1.1.1"),
 		},
+		&cli.StringFlag{
```

Comment:
I wasn't sure whether to add this option to `run.go` or `main.go`.  And I selected `run.go` .
In the containerd project, it is `containerd/cmd/ctr/commands/commands.go`, 
but it is only used as an option for `ctr run`.

Reply:
This should be in run.go, but the flag syntax needs to be `--security-opt seccomp=<PROFILE>`